protoc-gen-go-bbs/bbs.go

```diff
@@ -0,0 +1,719 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"slices"
+	"strings"
+	"text/template"
+
+	"google.golang.org/protobuf/compiler/protogen"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/types/descriptorpb"
+)
+
+type bbsGenerateHelperInterface interface {
+	genCopysafeStruct(g *protogen.GeneratedFile, msg *protogen.Message)
+	genToProtoMethod(g *protogen.GeneratedFile, msg *protogen.Message)
+	genFromProtoMethod(g *protogen.GeneratedFile, msg *protogen.Message)
+	genToProtoSliceMethod(g *protogen.GeneratedFile, msg *protogen.Message)
+	genFromProtoSliceMethod(g *protogen.GeneratedFile, msg *protogen.Message)
+	genFriendlyEnums(g *protogen.GeneratedFile, msg *protogen.Message)
+	genAccessors(g *protogen.GeneratedFile, msg *protogen.Message)
+	genEqual(g *protogen.GeneratedFile, msg *protogen.Message)
+}
+type bbsGenerateHelper struct{}
+
+var helper bbsGenerateHelperInterface = bbsGenerateHelper{}
+
+func getUnsafeName(g *protogen.GeneratedFile, ident protogen.GoIdent) string {
+	return g.QualifiedGoIdent(ident)
+}
+
+func getCopysafeName(g *protogen.GeneratedFile, ident protogen.GoIdent) (string, bool) {
+	unsafeName := getUnsafeName(g, ident)
+	return strings.CutPrefix(unsafeName, *prefix)
+}
+
+func getFieldName(goName string) string {
+	result := goName
+	// if strings.Contains(strings.ToLower(result), "lrp") {
+	// 	result = strings.Replace(goName, "Lrp", "LRP", -1)
+	// }
```

Comment:
uncomment or remove?

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)